setuptools/compat/py39.py

```diff
@@ -0,0 +1,9 @@
+import sys
+
+# Explicitly use the ``"locale"`` encoding in versions that support it,
+# otherwise just rely on the implicit handling of ``encoding=None``.
+# Since all platforms that support ``EncodingWarning`` also support
+# ``encoding="locale"``, this can be used to suppress the warning.
+# However, please try to use UTF-8 when possible
+# (.pth files are the notorious exception: python/cpython#77102, pypa/setuptools#3937).
+LOCALE_ENCODING = "locale" if sys.version_info >= (3, 10) else None
```

Comment:
@abravalheri  Should this comment be the variable's docstring? So it can be picked-up by editors and the likes? For example:
![image](https://github.com/pypa/setuptools/assets/1350584/bb5beebe-0743-431f-877f-0e1623886a0f)

Reply:
Initially I thought about doing that, but then I changed it back, because this module is not really targeting external users (it is not really part of the public API...)